Torch.Server/app.config

```diff
@@ -1,3 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8"/></startup></configuration>
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8" /></startup>
```

Comment:
Not part of my work, assuming it's an external update that hasn't been applied to master.